content/docs/command-reference/metrics/diff.md

```diff
@@ -7,10 +7,9 @@ commits in the <abbr>DVC repository</abbr>, or between a commit and the
 ## Synopsis
 
 ```usage
-usage: dvc metrics diff [-h] [-q | -v]
-                        [--targets [<path> [<path> ...]]]
-                        [-t <type>] [-x <path>] [-R] [--all]
-                        [--show-json] [--show-md] [a_ref] [b_ref]
+usage: dvc metrics diff [-h] [-q | -v] [--targets [<paths> [<paths> ...]]]
+                        [-R] [--all] [--show-json] [--show-md] [--no-path]
```

Comment:
same here - make sure we have all options described at least on some basic level

Reply:
They are 🙂